docs/how/delete-metadata.md

```diff
@@ -97,6 +97,21 @@ The start and end time fields filter on the `timestampMillis` field of the times
 - `ddddddddd` (e.g. `1684384045`): a unix timestamp
 - `min`, `max`, `now`: special keywords
 
+#### Undo-ing soft deletion of entities
+
+You can restore soft-deleted entities using the `undo_by_filter` command. This reverts the effect of a soft delete. 
+
+```shell
+# Restore (un-soft-delete) a single soft-deleted entity
+datahub delete undo_by_filter --urn "urn:li:dataset:(urn:li:dataPlatform:hive,fct_users_deleted,PROD)"
```

Comment:
command is `undo-by-filter`. `undo_by_filter` is method name

Reply:
@anshbansal fixed